packages/amplify-e2e-core/src/categories/storage.ts

```diff
@@ -610,9 +610,9 @@ export function updateS3AddTriggerNewFunctionWithFunctionExisting(cwd: string, s
   });
 }
 
-export function addS3StorageWithIdpAuth(projectDir: string): Promise<void> {
+export function addS3StorageWithIdpAuth(projectDir: string, testingWithLatestCodebase = false): Promise<void> {
```

Comment:
Shouldn't the default be true?

Reply:
nah. The patter is "false", see https://github.com/search?q=repo%3Aaws-amplify%2Famplify-cli%20testingWithLatestCodebase&type=code . and then we rely on cci/codebuild to set this up properly ...